### PR TITLE
add GCA fluid models

### PR DESCRIPTION
**Pull request by DavidRuhe**

### CODE REVIEW

File: cliffordlayers/models/gca_models_2d.py [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+    CliffordG3Conv2d,
+    CliffordG3ConvTranspose2d,
+    CliffordG3GroupNorm,
+    CliffordG3SiLU,
+)
+
+
+def get_activation(activation: str, channels: int) -> Callable:
+    if activation == "vsum":
+        return lambda x: torch.sigmoid(x.sum(-1, keepdim=True)) * x
```
Comment (rejuvyesh) [inline cliffordlayers/models/gca_models_2d.py]: Should probably make proper `nn.Module`s for these as they were quite useful.

**Review by rejuvyesh — APPROVED** [review]
Would be useful to document it a little so we can add them to auto-generated docs.

File: cliffordlayers/models/gca_models_2d.py (hunk 1 above)
Comment (DavidRuhe) [reply cliffordlayers/models/gca_models_2d.py]: Updated

**Comment by DavidRuhe** [comment]
Added docstrings.

**Pull request merged**